src/index.js

```diff
@@ -21,15 +22,25 @@ const useStyles = makeStyles(theme => ({
 const App = () => {
 
   const [page, setPage] = useState('app');
+  const [userId, setUserId] = useState(0);
```

Comment:
Let's store user object in this state: `[user, setUser] = useState({ username: 'anonymous', id: null  });`
Thus we can pass all user information at once to lower components.